userguide/getting-started/quick_start_with_kb_api.adoc

```diff
@@ -0,0 +1,1627 @@
+= Quick Start with the Kill Bill API
+
+This guide explains how to get familiar with the Kill Bill API by performing the following tasks:  
+
+. <<step1, Create a tenant>>
+. <<step2, Set up a simple catalog and plan>>
+. <<step3, Create an account>> 
+. <<step4, Add a payment method to the account>>
+. <<step5, Set up a subscription for the account>>
+. <<step6, View the generated invoice>>
+
+== Before You Begin
+
+Make sure you have installed Kill Bill and Kaui and launched them per the https://docs.killbill.io/latest/getting_started.html.html[_Getting Started_] guide. 
+
+== Additional Resources
+
+* https://killbill.github.io/slate[_API Reference_]
+
+* https://docs.killbill.io/latest/userguide_kaui.html[_Kaui Guide_] 
+
+* https://docs.killbill.io/latest/Kill-Bill-Glossary.html[_Kill Bill Glossary_]
+ 
+[step1]
+== Step 1. Create a Tenant
+
+Kill Bill supports multi-tenancy, where each https://docs.killbill.io/latest/Kill-Bill-Glossary.html#tenant[tenant^] has its own data, configuration, and so forth. This section explains how to create a tenant via the Kill Bill API.
+
+//All the code in this section is from the Tenants section in the API docs.
+
```

Comment:
I would also suggest adding a link to the API endpoint from the slate docs for each step. For example, to create a tenant, the [Create a Tenant](https://killbill.github.io/slate/?java#tenant-create-a-tenant) endpoint is used.

Reply:
Thanks for catching that we needed that!